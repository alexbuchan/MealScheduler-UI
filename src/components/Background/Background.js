import React from 'react';

const Background = () => {
  return (
    <div className='background'>
      <img
        src='https://productstewardshipinstitute.files.wordpress.com/2015/09/phonebookwithfingers-pg.jpg'
        alt='background-image'
        className='background-image'>
      </img>
    </div>
  );
}

export default Background;
