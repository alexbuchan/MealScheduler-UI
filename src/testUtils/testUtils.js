import React from 'react';
import { shallow } from 'enzyme';

export const componentSetup = (Component, props = {}, options = {}) => {
  const instance = shallow(<Component { ...props } />, options);
  return instance;
}

export const findByTestAttribute = (instance, attribute) => {
  const wrapper = instance.find(`[data-test="${attribute}"]`);
  return wrapper;
}