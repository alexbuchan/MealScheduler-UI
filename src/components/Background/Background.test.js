import Background from './Background';
import { componentSetup } from '../../testUtils/testUtils';

describe('Background', () => {
  let instance;
  
  beforeEach(() => {
    instance = componentSetup(Background);
  });

  it('Should render without errors', () => {
    expect(instance).toMatchSnapshot();
  });
});