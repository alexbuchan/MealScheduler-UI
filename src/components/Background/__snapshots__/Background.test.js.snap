// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background Should render without errors 1`] = `
<div
  className="background"
>
  <img
    alt="background-image"
    className="background-image"
    src="https://productstewardshipinstitute.files.wordpress.com/2015/09/phonebookwithfingers-pg.jpg"
  />
</div>
`;
